main.sh

```diff
@@ -1,12 +1,40 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
 # set environment variables from .env
 set -a
 source .env
 set +a
 
-if [ "$MODE" == "deploy" ]; then
-    gunicorn PLATER.services.app:APP -b 0.0.0.0:2304 -w 4 -k uvicorn.workers.UvicornWorker
-else
-    uvicorn PLATER.services.app:APP --host 0.0.0.0 --port 9747 --reload
-fi
+function webServer {
+    echo "Starting web server ... "
+    if [ "$MODE" == "deploy" ]; then
+        gunicorn PLATER.services.app:APP -b ${WEB_HOST}:${WEB_PORT} -w 4 -k uvicorn.workers.UvicornWorker
+    else
+        uvicorn PLATER.services.app:APP --host ${WEB_HOST} --port ${WEB_PORT} --reload
+    fi
+}
+
+function heartbeat {
```

Comment:
We can probably move _all_ of the heartbeat stuff to bash; we're only using Python to make a web request in a loop.